lib/augmentConfig.js

```diff
@@ -57,29 +57,33 @@ async function augmentConfigBasic(
 		filePath,
 	);
 
-	return absolutizePaths(augmentedConfig, configDir);
+	const cwd = stylelint._options.cwd || process.cwd();
```

Comment:
[suggestion] How about initializing `stylelint._options.cwd` in `createStylelint()` as below?
I think we can reduce the duplicate code: `stylelint._options.cwd || process.cwd()`.

```diff
diff --git a/lib/createStylelint.js b/lib/createStylelint.js
index bec7abc7..ffe0a045 100644
--- a/lib/createStylelint.js
+++ b/lib/createStylelint.js
@@ -22,12 +22,14 @@ const STOP_DIR = IS_TEST ? process.cwd() : undefined;
  * @returns {StylelintInternalApi}
  */
 function createStylelint(options = {}) {
+	const cwd = options.cwd || process.cwd();
+
 	/** @type {StylelintInternalApi} */
 	// @ts-expect-error -- TS2740: Type '{ _options: LinterOptions; }' is missing the following properties from type 'InternalApi'
-	const stylelint = { _options: options };
+	const stylelint = { _options: { ...options, cwd } };
 
 	stylelint._extendExplorer = cosmiconfig('', {
-		transform: augmentConfig.augmentConfigExtended(options.cwd),
+		transform: augmentConfig.augmentConfigExtended(cwd),
 		stopDir: STOP_DIR,
 	});
 

```

Reply:
Done in 328351e